### PR TITLE
Manoj comments

### PR DESCRIPTION
**Pull request by danielsuo**
NOTE: Please don't merge yet. Need to update HTML files.

### CODE REVIEW

**Comment by danielsuo** [comment]
Latest tutorial HTMLs added.

File: assets/css/application.css [hunk 1]
```diff
@@ -19,10 +19,6 @@ html, body {
 
 }
 
-.nav-link {
-  cursor: pointer
```
Comment (manojneuro) [inline assets/css/application.css]: I'm not sure if this is deleted by design.

**Comment by manojneuro** [comment]
@danielsuo I suspect an older index file is being used in the PR. A lot of the enhancements in the current file are being deleted.

**Comment by danielsuo** [comment]
Should be fine

**Comment by danielsuo** [comment]
You'll have to be more specific; which enhancements and which files? The changes to the website directory are fairly limited and I just pulled the latest from the tutorials repository.

**Comment by manojneuro** [comment]
The file tutorials/index.md seems to be older in the PR than the current tutorials/index.md

**Comment by danielsuo** [comment]
I see now. That's really weird. When I sent out the pull request, I didn't see all those additional changes. Let me fix.

**Comment by danielsuo** [comment]
Updated a few more things

**Comment by manojneuro** [comment]
@danielsuo The new ISC html is still not showing up.

**Comment by danielsuo** [comment]
Hm. But the diffs show tutorials was pulled to the latest commit...

https://github.com/brainiak/brainiak-tutorials/commit/a882201be919fc35ace70fe1369f21b63f0bc4d7

Does that have the latest html you expected?

**Comment by manojneuro** [comment]
Ok now I see it. Must be a cache issue.

**Comment by danielsuo** [comment]
Great!

**Pull request merged**